proto/catalog.proto

```diff
@@ -95,6 +95,15 @@ message Table {
     INDEX = 3;
     INTERNAL = 4;
   }
+  message TableVersion {
+    // The version number, which will be 0 by default and be increased by 1 for
+    // each schema change in the frontend.
+    uint64 version = 1;
```

Comment:
Previously we decided to use DDL epoch as the version in the design doc. But it seems there's no benefit to doing this. Simply using a per-table sequence works well and can be a pure-frontend thing.